README.md

```diff
@@ -383,6 +383,7 @@ productType,name.en,variantId
     -t, --template <file>         CSV file containing your header that defines what you want to export
     -o, --out <file>              Path to the file the exporter will write the resulting CSV in
     -j, --json                    Export in JSON format
+    -x, --xlsx                    Export in XLSX format
```

Comment:
I totally agree. But this feature is getting more urgent so I did not want to add other changes which are not necessary now - with this change we would break backward compatibility so it would need more time to track all systems which can be affected by this change.

If you agree, I would do that in another refactor.